solution/_templates/scripts/get-seed-data/clear-tables.sql

```diff
@@ -0,0 +1,15 @@
+delete from resources_abstractcategory;
+delete from resources_abstractlocation;
+delete from resources_abstractresource; 
+delete from resources_abstractresource_locations; 
+delete from resources_abstractresource_related_resources; 
+delete from resources_category; 
+delete from resources_federalregisterdocument; 
+delete from resources_federalregisterdocumentgroup; 
+delete from resources_resourcesconfiguration; 
+delete from resources_section; 
+delete from resources_subcategory; 
+delete from resources_subpart; 
+delete from resources_supplementalcontent; 
+delete from search_synonym; 
+delete from search_synonym_synonyms; 
```

Comment:
I am not comfortable with just having a script that deletes the data to be updated on its own. I think this needs to be tightened up a little.
1. put something in place that says "are you sure and this is what this script will do" 
2. Add dump command before the delete right in here so its always backing up if you run this.
3. Put a check in here that checks the environment so that this can only be run on dev and not on prod or val. 
4. Write tests